Rakefile

```diff
@@ -48,26 +60,42 @@ task :benchmark do
   average_request = TestSignableRequest.new(verb: 'PUT', request_url: '/', body: average_body)
   huge_request = TestSignableRequest.new(verb: 'PUT', request_url: '/', body: huge_body)
 
```

Comment:
Perhaps we should have some reqs with query parameters for V2 as well?

Reply:
Added. Thought about adding a normal size query string and a huge query string but a huge query string doesn't seem very likely (whereas a huge body seems relatively likely).

Reply:
Don't think we need the V1 with query case tho, since V1 does not do anything with query 😉

Reply:
Removed